### PR TITLE
FTP-1384:navbar

### PR DESCRIPTION
**Pull request by HamidAli111**
Screenshot 2022-10-11 at 4 20 07 PM

### CODE REVIEW

File: components/navbar/Navbar.stories.tsx [hunk 1]
```diff
@@ -12,8 +12,26 @@ const Template: ComponentStory<typeof Navbar> = (args) => <Navbar {...args} />;
...
+      title: 'Bag',
+      link: '/bag',
+      icon: BagIcon,
+    },
+    {
+      title: 'Menu',
+      link: '/menu',
+      icon: MenuIcon,
+    },
+  ],
```
Comment (UmmarHamid) [inline components/navbar/Navbar.stories.tsx:36]: Is this required? These seem similar to default links

File: components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    icon: MenuIcon,
+    link: '/menu',
+  },
+];
+
+function Navbar({ links = defaultLinks }: navbarType) {
+  return (
+    <div className="flex pt-20 justify-around items-center bg-white-regular rounded-t-xl m-10 shadow-primary max-w-screen-sm h-116 mt-10">
+      {links.map((item) => (
+        <a key={item.title} href={item.link}>
```
Comment (skippednote) [inline components/navbar/Navbar.tsx:33]: please add the title on the link

File: pages/testingpage.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (skippednote) [inline pages/testingpage.tsx:1]: Let's remove the page

File: components/navbar/Navbar.stories.tsx (hunk 1 above)
Comment (skippednote) [reply components/navbar/Navbar.stories.tsx:36]: Let's remove links

**Review by skippednote — CHANGES_REQUESTED** [review]

File: @types/navbar.types.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+type navbarType = {
+  links?: { title: string; link: string; icon: string }[];
+};
+export default navbarType;
```
Comment (skippednote) [inline @types/navbar.types.ts:4]:
```suggestion
export type NavbarLink = { title: string; link: string; icon: string };
type Navbar = {
  links?: NavbarLink[];
};
export default Navbar;
```

**Review by skippednote — CHANGES_REQUESTED** [review]

**Pull request merged**